### DISCUSSION TITLE
v.5.1.0

### DISCUSSION THREAD

**Discussion by DanRobo76** [DanRobo76/VMC-HELTY-FLOW#16] [Generale]
- Permette di selezionare le entità di: Temperatura (Interna ed Esterna), Umidità (Interna ed Esterna), Anidride Carbonica CO₂ (Interna) e Dispositivo Mobile, disponibili nel sistema di Home Assistant (H.A.), includendo anche le entità provenienti da fonti Meteo, per ottenere una maggiore precisione e affidabilità rispetto ai sensori di Temperatura e Umidità integrati nella VMC;
- Avvisa l'utente tramite notifica sul Dispositivo Mobile, in caso di Avvio, Arresto, Errori o cambio filtro della VMC;
- Corretto l'errore Time-out;
- Migliorato il codice e le Logiche di Funzionamento;
- Scheda Interfaccia Release 5.1.0 del 15/12/2024
- Automazione Release 5.1.0 del 15/12/2024
- Configurazione Release 5.1.0 del 15/12/2024

- CLONA VMC HELTY FLOW PLUS Release V2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/DanRobo76/VMC-HELTY-FLOW/releases/tag/v.5.1.0'>v.5.1.0</a>.</em>

**Comment by pet1972s** [comment]
During the last few days, Home Assistant notified several times a new version in the repository. But on the site there is still version 5.1.0, the same one I have installed. What could be the reason?

**Reply by DanRobo76** [reply]
I have no idea, it hasn't happened to me yet!

**Comment by erionberliu** [comment]
Buongiorno Danilo,
Innanzitutto grazie mille per questo capolavoro, avevo provato a domandare anche a Helty come fare per poter controllare da remoto le unità VMC ma è sempre stato risposto fosse impossibile. Ora che da circa un mese ho intrapreso la strada di HomeAssistant, ho trovato questo tuo capolavoro.
Dunque, ho provato a seguire passo per passo la tua guida ma ad un certo punto mi blocco e non capisco dove sbaglio.
Sostanzialmente dispongo di 3 unità, soggiorno, camera e cucina.
L'unità della "cucina" sono riuscito a configurarla senza alcun problema, quando invece provo a clonare, pure inserendo l'IP soggiorno (statico)  e nome unità "soggiorno", clicco "avvio clona vmc" , risulta = vmc clonata, e mi rimanda nella cartella di packages dove in teoria mi dovrei trovare due nuovi file "automazione" e "interfaccia" relativa alla nuova unità, per poi copiare ed incollare il contenuto nella nuova automazione e nuova interfaccia. Il problema è che non trovo questi due nuovi file. 
Sto sbagliando qualcosa? 
Attualmente l'interfaccia è la seguente:  
![vmc clonata](https://github.com/user-attachments/assets/7ae57410-2ee0-4987-9b85-8e246677c876)
Ti sarei grato se potessi darmi una mano.
Cordialmente
Erion

**Reply by DanRobo76** [reply]
devi seguire attentamente le istruzioni, creando il percorso giusto:

3 Verificare la presenza della cartella packages all'interno della cartella config del Server Home Assistant; se non è presente, creare l'intero percorso ovvero \\192.168.1.xxx\config\packages\vmc_helty_flow\master

Fammi sapere!

**Reply by erionberliu** [reply]
Attualmente io mi ritrovo solo con questo percorso. Ho provato nuovamente a clonare ma non mi crea la cartella relativa alla nuova entità "soggiorno"
![Screenshot 2025-02-14 135726](https://github.com/user-attachments/assets/98ade79d-c08b-4ea4-9dea-2b0545d22b6f)

**Comment by DanRobo76** [comment]
come puoi vedere non è uguale a: \packages\vmc_helty_flow\master

**Comment by erionberliu** [comment]
ti ringrazio moltissimo.. sono riuscito a configurare tutto.
Grazie e ancora complimenti per il bel lavoro.

**Reply by DanRobo76** [reply]
se vuoi offrirmi un caffe :)
